<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symularor UAR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutParams">
      <item>
       <widget class="QGroupBox" name="groupPID">
        <property name="title">
         <string>Regulator PID</string>
        </property>
        <layout class="QFormLayout" name="formLayoutPID">
         <item row="0" column="0">
          <widget class="QLabel" name="labelK">
           <property name="text">
            <string>K:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="spinK">
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelTi">
           <property name="text">
            <string>Ti:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="spinTi">
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelTd">
           <property name="text">
            <string>Td:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="spinTd"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupGen">
        <property name="title">
         <string>Generator</string>
        </property>
        <layout class="QGridLayout" name="gridLayoutGen">
         <item row="0" column="0">
          <widget class="QLabel" name="lT">
           <property name="text">
            <string>Tryb:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboTryb">
           <item>
            <property name="text">
             <string>Prostokątny</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sinusoidalny</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lA">
           <property name="text">
            <string>Amplituda:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="spinAgen">
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lTT">
           <property name="text">
            <string>T sym (TT):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinTT">
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="lTRZ">
           <property name="text">
            <string>T real (TRZ):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="spinTRZ">
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lS">
           <property name="text">
            <string>Offset (S):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="spinSgen"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="lP">
           <property name="text">
            <string>Wypełnienie:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="spinPgen">
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenARX">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Parametry ARX</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayoutCharts"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutBtns">
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>RESET</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblInt">
        <property name="text">
         <string>Interwał (ms):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinInterval">
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveConfig">
        <property name="text">
         <string>Zapisz JSON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoadConfig">
        <property name="text">
         <string>Wczytaj JSON</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
